### DISCUSSION TITLE
Is there a way to apply the modification when the module of node_modules is modified?

### DISCUSSION THREAD

**Discussion by ITSWYoo** [vercel/next.js#15044] [Help] [answered]
Is there a way to apply the modification when the module of node_modules is modified?

like this
```js
// node_modules/something
~~~
console.log("test") // i add console to test
~~~
```
It is called in the browser, but not in the next.js server. (yarn dev)

**Comment by jamesmosier** [comment, marked as answer]
That would get overwritten anytime you install your node_modules (for example on a server) so it would not be reliable to do this.

**Reply by ITSWYoo** [reply]
@jamesmosier `next dev` overwritten?
